src/Resources/config/doctrine/BillingData.orm.xml

```diff
@@ -1,19 +1,23 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping
+                                      http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+
     <mapped-superclass name="Sylius\InvoicingPlugin\Entity\BillingData" table="sylius_invoicing_plugin_billing_data">
         <id name="id" column="id" type="integer">
             <generator strategy="AUTO" />
         </id>
 
-        <field name="firstName" column="first_name" />
-        <field name="lastName" column="last_name" />
-        <field name="company" column="company" nullable="true" />
-        <field name="street" column="street" />
-        <field name="city" column="city" />
-        <field name="postcode" column="postcode" />
-        <field name="countryCode" column="country_code" />
-        <field name="provinceCode" column="province_code" nullable="true" />
-        <field name="provinceName" column="province_name" nullable="true" />
+        <field name="firstName" column="first_name" type="string" />
+        <field name="lastName" column="last_name" type="string" />
+        <field name="company" column="company" type="string" nullable="true" />
+        <field name="street" column="street" type="string" />
+        <field name="city" column="city" type="string" />
+        <field name="postcode" column="postcode" type="string" />
+        <field name="countryCode" column="country_code" type="string" />
+        <field name="provinceCode" column="province_code" type="string" nullable="true" />
+        <field name="provinceName" column="province_name" type="string" nullable="true" />
```

Comment:
Why did you want to normalize mappings by adding these default values that are not necessary? The same applies to changes in other mappings

Reply:
How come column **type** is not a necessary field? Why rely on doctrine to have expected defaults instead of setting correct type explicitly?

There is difference in how source code is written at `App\` level and `Sylius\Sylius` level.
This is as much documentation as correct way to write entity mappings.

Besides, this is the way all sylius/sylius mappings are done.
https://github.com/Sylius/Sylius/blob/master/src/Sylius/Bundle/CoreBundle/Resources/config/doctrine/model/Order.orm.xml

Reply:
The default value of **type** is `string`, so IMO there is no need to add it here, as they weren't defined before, especially in the PR with the title `Fix getId/id return type for entities`, if you would like to change that, it should be done in the separate PR

Reply:
Doctrine mappings and entity fields types are quite related. Which issue will be solved by moving this to separate PR?

Please ask for 3rd opinion within the core team. Personally I don't see what there to discuss, `sylius/sylius` defines explicit doctrine mappings and that's how it should be done.

If you have different opinion on the matter -- please open RFC at `sylius/sylius` to remove string `type` definition from mappings.

Reply:
@diimpp you've opened a PR with fixing getId return types, but you've also refactored other methods and the corresponding mapping, this refactoring should be done in a separate PR.

Reply:
And fyi, I've already asked @Zales0123 and @lchrusciel for other opinions 😃